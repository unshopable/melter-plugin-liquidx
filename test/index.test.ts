import run from './helpers/run';

it('should transform Liquid to LiquidX', async () => {
  const stats = await run();

  expect(stats).toHaveProperty('assets.0');
  expect(stats.assets[0].content.toString()).toMatchSnapshot();
});
