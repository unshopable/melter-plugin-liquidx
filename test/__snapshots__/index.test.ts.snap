// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should work 1`] = `"{% capture ButtonChildren %}Hello, World!{% endcapture %}{% render 'Button', children: ButtonChildren %}"`;
